addons/point_of_sale/static/src/js/models.js

```diff
@@ -162,7 +164,26 @@ class PosGlobalState extends PosModel {
         this.uom_unit_id = uom_id[1];
     }
 
+    shouldInvoiceNewTab(actionRecord) {
+        return isIOS() && actionRecord.type === "ir.actions.report" && actionRecord.report_type === "qweb-pdf";
```

Comment:
Does the isIOSApp() check also exist in 16.0 ?

Reply:
I added `isIOSApp` in the check.